src/ZendCodingStandard/Sniffs/Files/CopyingSniff.php

```diff
@@ -0,0 +1,92 @@
+<?php
```

Comment:
I'm assuming this file and class should change to `CopyrightSniff` with the suggested `COPYRIGHT.md` change.